layouts/partials/footer.html

```diff
@@ -10,7 +10,7 @@
                 Twitter
             </div>
         </a>
-        <a class="footer-social-link" href="https://slack.paketo.io/">
+        <a class="footer-social-link" href="https://paketobuildpacks.slack.com/">
```

Comment:
```suggestion
        <a class="footer-social-link" href="https://join.slack.com/t/paketobuildpacks/shared_invite/zt-2jayv12ro-eTP8AtcmvyIpEtlANfIb~g">
```

I have been unable to hunt down the vm that is running that end point however I have created a link that will only expire after 400 uses